### ISSUE TITLE
Doesn't appear to be using ebpf

### ISSUE THREAD

**Issue by sbahra** [mranv/ebpf-file-monitor#6]
Expected? Seems like it is a placeholder for future plans and just uses a simple inotify-based loop.

**Comment by bobbintb** [comment]
Yeah, this seems to be just a boiler-plate inotify example. Nothing to do with bpf.
